pkg/users/accounts-list.js

```diff
@@ -295,31 +346,31 @@ const GroupsList = ({ groups, accounts }) => {
                                         </Label>
                                     );
                                 })}
-                        <Button key="more" className="group-more-btn" isInline variant='link' onClick={() => setIsExpanded(!isExpanded)}>
-                            {cockpit.format(_("$0 more..."), groups.length - 3)}
-                        </Button>
+                        {filtered_groups.length > 3 && <Button key="more" className="group-more-btn" isInline variant='link' onClick={() => setIsExpanded(!isExpanded)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-161842-791100a2-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)